src/import/chips/p9/procedures/hwp/perv/p9_sbe_lpc_init.C

```diff
@@ -7,6 +7,7 @@
 /*                                                                        */
 /* Contributors Listed Below - COPYRIGHT 2015,2018                        */
 /* [+] International Business Machines Corp.                              */
+/* COPYRIGHT 2020 Raptor Engineering, LLC                                 */
```

Comment:
Is this form of the copyright header right?

Reply:
Probably not, actually.  The whole header seems odd, it looks auto-generated but we don't have access to the tooling to regenerate it.  If there's a preferred format I'll update.